lti_consumer/static/js/xblock_lti_consumer.js

```diff
@@ -46,7 +46,13 @@ function LtiConsumerXBlock(runtime, element) {
                         $(o.closeButton).on('keydown', function (e) {
                            if (e.which === 9) {
                                e.preventDefault();
-                               $(modal_id).find('iframe')[0].contentWindow.focus();
+                               // This is a workaround due to Firefox triggering focus calls oddly.
+                               setTimeout(
```

Comment:
this can be consolidated into 3 lines?
setTimeout(function() {
    $modal.find('iframe')[0].contentWindow.focus();
}, 1);